streaming/src/main/scala/org/apache/spark/streaming/scheduler/ReceiverTracker.scala

```diff
@@ -137,8 +141,24 @@ class ReceiverTracker(ssc: StreamingContext, skipReceiverLaunch: Boolean = false
     }
     receiverInfo(streamId) = ReceiverInfo(
       streamId, s"${typ}-${streamId}", receiverActor, true, host)
+    logInfo(s"Receiver for stream ${streamId} from ${sender.path.address} " +
+      s"attempts to register.")
+  }
+
+  /** Report a starting receiver */
+  private def startingReceiver(
```

Comment:
I'm not sure is it make sense if we can delete the related code of sending `RegisterReceiver` in receiver's preStart(), only send this message in onReceiverStart(). So this message will only send once.